docs/guides/introduction/about.md

```diff
@@ -1 +1,23 @@
-# About Daffodil
+# Introduction
+
+## What is Daffodil?
+
+Daffodil is a Typescript framework for building headless ecommerce stores. It is built on top of Angular and provides a suite of composable packages that allow you to build complicated ecommerce stores.
```

Comment:
`complex` instead of complicated?
```suggestion
Daffodil is a Typescript framework for building headless ecommerce stores. It is built on top of Angular and provides a suite of composable packages that allow you to build complicated ecommerce stores.
```
```suggestion
Daffodil is an Angular framework for building headless ecommerce stores. It provides a suite of composable packages to help you build complex ecommerce solutions.
```

Typescript isn't a framework?

Repeating `ecommerce stores` back to back sounds redundant.

Reply:
Daffodil is a Typescript framework. Typescript is modifying the word framework.

Reply:
I fixed 

> complex instead of complicated? 

in ac503f79c

> Daffodil is a frontend Angular framework for building ecommerce stores

Angular itself is a framework, so this phrasing sounds strange. It think this would be accurate "Daffodil is a frontend framework for building ecommerce stores".